docs/conversion_examples_gallery/behavior/deeplabcut.rst

```diff
@@ -0,0 +1,27 @@
+DeepLabCut data conversion
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Convert DeepLabCut imaging data to NWB using :py:class:`~neuroconv.datainterfaces.behavior.deeplabcut.deeplabcutdatainterface.DeepLabCutInterface`.
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from neuroconv import DeepLabCutInterface
+    >>>
+    >>> file_path = BEHAVIOR_DATA_PATH / "DLC" / "m3v1mp4DLC_resnet50_openfieldAug20shuffle1_30000.h5"
+    >>> config_file_path = BEHAVIOR_DATA_PATH / "DLC" / "config.yaml"
+    >>> subject_name = "ind1"
+    >>>
+    >>> interface = DeepLabCutInterface(file_path=file_path, config_file_path=config_file_path, subject_name=subject_name, verbose=False)
+    >>> metadata = interface.get_metadata()
+    >>> # For data provenance we add the time zone information to the conversion
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
+    >>> metadata.get("NWBFile", dict()).update(session_start_time=session_start_time)
+    >>> # Choose a path for saving the nwb file and run the conversion
+    >>> nwbfile_path = f"{path_to_save_nwbfile}"
+    >>> interface.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata)
+    >>> # If the conversion was successful this should evaluate to ``True`` as the file was created.
+    >>> Path(nwbfile_path).is_file()
```

Comment:
```suggestion
    >>> interface.run_conversion(nwbfile_path=path_to_save_nwbfile, metadata=metadata)
    >>> # If the conversion was successful this should evaluate to ``True`` as the file was created.
    >>> Path(path_to_save_nwbfile).is_file()
```

Reply:
That works as well, the previous format was trying to make explicit that you can pass the path as a string which I thought would be the simplest way to most people. But from there to "path_to_save_nwbfile` I guess there is not a lot of difference between these two. 

I committed your suggestion.